src/store/modules/requests/actions.js

```diff
@@ -0,0 +1,11 @@
+export default {
+  contactCoach(context, payload) {
+    const newRequest = {
+      id: new Date().toISOString(),
+      coachId: payload.coachId,
+      userEmail: payload.email,
+      message: payload.message,
+    };
+    context.commit('addRequest', newRequest)
+  }
+}
```

Comment:
@HVRolim newline no final do arquivo
```suggestion
}

```